README.markdown

```diff
@@ -5010,6 +5011,17 @@ This function was first introduced in the `v0.5.0rc1` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+ngx.req.set\_cong
+-----------------
+
+**syntax:** ngx.req.set_cong(cong_name)*
+
+**context:** *access_by_lua&#42;, content_by_lua&#42;*
+
+Set TCP congestion control for each request connection. It only works on Linux platform and if target TCP congestion is not exists, it will generate a warning log.
```

Comment:
we should use the term "congestion control algorithm" instead of "congestion control". The latter is confusing.

Also "is not exists" is not correct English. We say "does not exit".

And we need the word `the` before `target`.

Reply:
Also, we should list some typical (if not all) algorithm names allowed by modern Linux kernels.